onnxruntime/contrib_ops/cpu/inverse.cc

```diff
@@ -41,8 +41,8 @@ struct Inverse::ComputeImpl {
     const auto* input_data = input->Data<T>() + batch_offset;
     auto* output_data = output->MutableData<T>() + batch_offset;
 
-    Eigen::Map<const MatrixT<T>> input_matrix(input_data, rows, cols);
-    Eigen::Map<MatrixT<T>> output_matrix(output_data, rows, cols);
+    Eigen::Map<const MatrixT<T>> input_matrix(input_data, gsl::narrow_cast<size_t>(rows), gsl::narrow_cast<size_t>(cols));
```

Comment:
As suggested by Ed, gsl::narrow is prefered to gsl::narrow_cast when we want to check the range and throw exceptions when needed.